### DISCUSSION TITLE
[Feature request] Card Templates

### DISCUSSION THREAD

**Discussion by marcus-gordon-thomas** [mgmeyers/obsidian-kanban#825] [Ideas & Feature Requests]
Can we implement card templates? 
I like to keep track of when a task is created and when I move it around, and I always have a section in my cards for creation date, begun date, and finished date. If I could make a template card, then it would be faster to create tasks and keep them streamlined.

If this is already possible and someone could help point me in the right direction for setting it up, that would also be swell! :)

**Comment by MicaelJarniac** [comment]
I believe it may be possible to do it with a plugin like [QuickAdd](https://github.com/chhoumann/quickadd), as you can see in this video by the plugin creator:

https://youtu.be/gYK3VDQsZJo?t=1656

It is not an ideal solution, but it could do the trick for the time being.
